docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,2182 @@
+{
```

Comment:
<p>Would you mind if I put a gzipped version of the CSV into our <code>ibis-tutorial-data</code> GCS bucket to avoid the download and the extra code it requires? This also amplifies the effect of caching :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I think I actually slightly lean towards keeping this as is. I think the current method is more realistic and is easier for the reader to empathize with. With the GCS it would feel more like a cute example than a real task. The current method isn't <strong>that</strong> long. If you feel strongly though then I can update.</p>

Reply:
<p>Nope, we can leave it as is!</p>